### PR TITLE
name HEAD files as *wip

### PR DESCRIPTION
**Pull request by duglin**
Signed-off-by: Doug Davis <dug@us.ibm.com>

### CODE REVIEW

File: amqp-protocol-binding.md [hunk 1]
```diff
@@ -224,7 +224,7 @@ content-type: application/json; charset=utf-8
 
 ----------- application-properties -----------
 
-cloudEvents:specversion: 1.0
+cloudEvents:specversion: 1.x-wip`
```
Comment (Vlaaaaaaad) [inline amqp-protocol-binding.md:227]:
```suggestion
cloudEvents:specversion: 1.x-wip
```

File: amqp-protocol-binding.md [hunk 2]
```diff
@@ -286,7 +286,7 @@ content-type: application/cloudevents+json; charset=utf-8
 ------------- application-data --------------------------
 
 {
-    "specversion" : "1.0",
+    "specversion" : "1.x-wip`
```
Comment (Vlaaaaaaad) [inline amqp-protocol-binding.md:289]:
```suggestion
    "specversion" : "1.x-wip",
```

File: amqp-protocol-binding.md (hunk 1 above)
Comment (duglin) [reply amqp-protocol-binding.md:227]: fixed - good eye

File: amqp-protocol-binding.md (hunk 2 above)
Comment (duglin) [reply amqp-protocol-binding.md:289]: fixed

**Comment by duglin** [comment]
@Vlaaaaaaad thanks for the review.
Merging since the format was agreed on the 5/21 call and it's just a syntax change.

**Pull request merged**